p5js/FaceApi/FaceApi_Image_Landmarks/sketch.js

```diff
@@ -0,0 +1,116 @@
+let faceapi;
+let img;
+let detections;
+
+// by default all options are set to true
+const detection_options = {
+    withFaceLandmarks: true,
+    withFaceExpressions: false,
+    withFaceDescriptors: false,
+}
+
+function preload(){
+    img = loadImage('assets/frida.jpg')
+}
+
+function setup() {
+    createCanvas(200, 200);
+    img.resize(width, height);
+
+    faceapi = ml5.faceApi(detection_options, modelReady)
```

Comment:
Are these optional and are there defaults? Or always required? I might suggest losing the `with` or `Face` if it's still clear enough? An old rule of thumb we always had back in the golden Processing days was two words only for methods and variables in the API 🤓 

This might be too bold though:

```javascript
const detection_options = {
    landmarks: true,
    expressions: false,
    descriptors: false,
}
```

Reply:
@shiffman - thanks for the comments!

Re licensing: 
+ the face-api.js is MIT licensed: https://github.com/justadudewhohacks/face-api.js/blob/master/LICENSE  so we should be ok!

Re options:
+ Yes the options are optional 😂 - by default we return all the data, but it runs a bit slower that way. So selectively choosing which data to return increases performance. So in this case people have to specify in the options which data to turn "off". 
+ I kept the **with** in `withLandmarks" etc since it refers to the way that the face-api.js refers to adding those data as part of the face detections. I think it is ok to also leave out the "with", but maybe it is helpful to make that conceptual linkage across the libraries? Also semantically to say "detections with landmarks"?

Reply:
changed options to →

```
{
    withLandmarks: true,
    withExpressions: true,
    withDescriptors: true,
}
```